src/Utils.ts

```diff
@@ -0,0 +1 @@
+export const UUIDRegex = /[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}/;
```

Comment:
This doesn't need to be exported, it just needed to be a const not re-instantiated for each guard invocation...